### ISSUE TITLE
move and mark as read

### ISSUE THREAD

**Issue by n0cturnuz** [opto/nostalgy-xpi#145]
Feature request

would it be possible to add an option that when moving a message it get marked as read as well?

Thnx

**Comment by opto** [comment]
yes, the main problem is we do not have so many letters, many are captured by TB. Maybe something like ctrl or shift S can work.

**Comment by n0cturnuz** [comment]
i mean just as a checkbox e.g. [ ] on move, mark as read
not as a special key (or key combo)

**Comment by opto** [comment]
in my TB, M(R) set messages read or unread.

So is it a workaround to first set message( un)read, then move by S?

**Comment by opto** [comment]
so MS: move as unread, RS: move as read
